mutaremalcolm/calorie_counter_app
# Pull Request: Add Sidebar Component to Dashboard

## Description
This PR introduces a sidebar component to the dashboard. The sidebar includes links for easy navigation to various sections of the application. 

## Changes Made
- Added a new Sidebar component to the dashboard.
- Implemented routes for all links in the sidebar for seamless navigation.

## How to Test
1. Navigate to the dashboard.
2. Verify that the sidebar is displayed correctly.
3. Click on each link in the sidebar and ensure that it routes to the corresponding section of the application.

## Acceptance Criteria
- The sidebar is visually integrated into the dashboard layout.
- All sidebar links function correctly and route to the intended pages.
